### PR TITLE
add test for test-logging-tx-observer

### PR DESCRIPTION
**Pull request by tunedev**
## Checklist
 - [ ]  A link to the issue/user story that the pull request relates to
 - [ ]  How to recreate the problem without the fix
 - [ ]  Design of the fix
 - [ ]  How to prove that the fix works
 - [ ]  Automated tests that prove the fix keeps on working
 - [ ]  Documentation - any JSDoc, website, or Stackoverflow answers?

## Issue/User story

Partially addresses #1606

## Steps to Reproduce

1.
2.
3.
4.

## Existing issues

- [ ] [Stack Overflow issues](http://stackoverflow.com/tags/hyperledger-caliper)
- [x] [GitHub Issues](https://github.com/hyperledger/caliper/issues)
- [ ] [Discord history](https://discord.com/channels/905194001349627914/941417677778473031)

## Validation of the fix

After this PR: Running the tests in caliper-core the listed %stmts in the code coverage report should tally with the below listed

- caliper-core/lib/worker/tx-observers/logging-tx-observer.js: 100

Before the PR: Running the tests in caliper-core the listed %stmts in the code coverage report the below listed was what was there before

- caliper-core/lib/worker/tx-observers/logging-tx-observer.js: 0

### CODE REVIEW

File: packages/caliper-core/test/worker/tx-observers/logging-tx-observer.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        logStub.resetHistory();
+    });
+
+    after(() => {
+        mockery.deregisterAll();
+        mockery.disable();
+    });
+
+    describe('On initialization', () => {
+        it('should set up the logger with default settings if no options are provided', () => {
```
Comment (davidkel) [inline packages/caliper-core/test/worker/tx-observers/logging-tx-observer.js:61]: Don't think this test proves anything as the logger will always exist

**Review by davidkel — CHANGES_REQUESTED** [review]
Sorry, needs a few changes here. It also breaks the build (see the build details in the PR and github actions). Please run the unit and integration tests before submitting the PR to avoid this

**Review by davidkel — CHANGES_REQUESTED** [review]
One last change

**Comment by tunedev** [comment]
true, would remove it now

**Review by davidkel — APPROVED** [review]
LGTM

**Pull request merged**